recipes/ignition-cmake/all/conanfile.py

```diff
@@ -30,15 +34,21 @@ def _configure_cmake(self):
         self._cmake = CMake(self)
         self._cmake.definitions["CMAKE_INSTALL_DATAROOTDIR"] = "lib"
         self._cmake.definitions["SKIP_component_name"] = False
-
         self._cmake.configure(source_folder=self._source_subfolder)
         return self._cmake
 
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
-        os.rename(glob.glob("ign-cmake*")[0], self._source_subfolder)
+        version_major = self.version.split(".")[0]
+        conan.tools.files.rename(
+             self, f"gz-cmake-ignition-cmake{version_major}_{self.version}",
+             self._source_subfolder
+            )
 
     def build(self):
+        version_major = tools.Version(self.version).major
+        ignition_cmake_dir = os.path.join(self.package_folder, "lib", "cmake", f"ignition-cmake{version_major}", "cmake2")
+        os. environ['IGNITION_CMAKE_DIR'] = f"{ignition_cmake_dir}"
```

Comment:
```suggestion
       env_build = RunEnvironment(self)
       env_build["IGNITION_CMAKE_DIR"] = f"{ignition_cmake_dir}"
       with tools.environment_append(env_build.vars): 
```
Try to use something more auto-contained then os.environ
https://docs.conan.io/en/latest/reference/build_helpers/run_environment.html#run-environment-reference

Reply:
applied manually. thanks for the hint